src/apps/pulse/components/Buy.tsx

```
@@ -0,0 +1,364 @@
+            <div style={{color: "#8A77FF", marginLeft: 5, fontSize: 12, lineHeight: 0.9}}>
+              ${props.walletPortfolioData?.result.data.total_wallet_balance.toFixed(2) || 0.00}
+            </div>
+          </div>
+        </div>
+      </div>
+  
+      {/* amounts */}
+      <div className="flex">
+        {["10", "20", "50", "100", "MAX"].map(item => {
+          return (
+            <div className="flex" style={{backgroundColor: "black", marginLeft: 10, width: 75, height: 30, borderRadius: 10}}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)